src/components/search/search.tsx

```diff
@@ -155,23 +90,29 @@ const Search: React.FunctionComponent<SearchProps> = ({
                         name="Snippet"
                         label="Code Snippets"
                         onToggle={checked => onCheckToggle(checked, 'Snippet')}
-                        checked={codeLevelFilters.includes('Snippet')}
+                        checked={
+                            !!allFilters.find(filt => filt.name === 'Snippet')
```

Comment:
small nitpick but I would have created a `const` called `isSnippetChecked` and reference it here.